system/ee/legacy/core/Config.php

```diff
@@ -547,7 +547,7 @@ public function divination($which)
             'password_lockout_interval',
             'require_ip_for_login',
             'require_ip_for_posting',
-            'require_secure_passwords',
+            'secure_passwords',
             'allow_dictionary_pw',
```

Comment:
@intoeetive if someone currently has require_secure_passwords turned on, are we migrating them to basic?

Reply:
This change was made by mistake, reverted it. Thank you for bringing it to my attention